mpas_analysis/ocean/histogram.py

```diff
@@ -0,0 +1,593 @@
+            control_filename = get_unmasked_mpas_climatology_file_name(
+                self.controlConfig, self.season, self.componentName, op='avg')
+            ds_control = xarray.open_dataset(control_filename)
+            base_directory = build_config_full_path(
+                self.controlConfig, 'output', 'histogramSubdirectory')
+            control_region_mask_filename = f'{base_directory}/histogram_' \
+                f'{control_run_name}_{self.regionName}_mask.nc'
+            ds_control_region_masks = xarray.open_dataset(
+                control_region_mask_filename)
+            control_cell_mask = ds_control_region_masks.regionCellMasks == 1
+            if self.weightList is not None:
+                control_weights_filename = \
+                    f'{base_directory}/histogram_{control_run_name}_' \
+                    f'{self.regionName}_weights.nc'
+                ds_control_weights = xarray.open_dataset(
+                    control_weights_filename)
+
+        if config.has_option(self.taskName, 'lineColors'):
+            lineColors = [config.get(self.taskName, 'mainColor')]
+        else:
+            lineColors = None
+        if config.has_option(self.taskName, 'obsColor'):
+            obsColor = config.get_expression(self.taskName, 'obsColor')
+            if lineColors is None:
+                lineColors = ['b']
+        else:
+            if lineColors is not None:
+                obsColor = 'k'
+
+        if config.has_option(self.taskName, 'lineWidths'):
+            lineWidths = [config.get(self.taskName, 'lineWidths')]
+        else:
+            lineWidths = None
+
+        title = main_run_name
+        if config.has_option(self.taskName, 'titleFontSize'):
+            titleFontSize = config.getint(self.taskName,
+                                          'titleFontSize')
+        else:
+            titleFontSize = None
+        if config.has_option(self.taskName, 'titleFontSize'):
+            axisFontSize = config.getint(self.taskName,
+                                         'axisFontSize')
+        else:
+            axisFontSize = None
+
+        if config.has_option(self.taskName, 'defaultFontSize'):
+            defaultFontSize = config.getint(self.taskName,
+                                            'defaultFontSize')
+        else:
+            defaultFontSize = None
+        if config.has_option(self.taskName, 'bins'):
+            bins = config.getint(self.taskName, 'bins')
+        else:
+            bins = None
+
+        yLabel = 'normalized Probability Density Function'
+
+        for index, var in enumerate(self.variableList):
+
+            fields = []
+            weights = []
+            legendText = []
+
+            var_name = f'timeMonthly_avg_{var}'
+
+            caption = f'Normalized probability density function for {var} ' \
+                      f'climatologies in {self.regionName.replace("_", " ")}'
```

Comment:
Could you add the season to the caption, too?  This might be redundant with the title but I think we've included it in other captions.

Reply:
No problem